pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -74,6 +74,11 @@
    * non-alphanumeric characters. */
   private static final Lex PINOT_LEX = Lex.MYSQL_ANSI;
 
+  // BABEL is a very liberal conformance value that allows anything supported by any dialect
+  private static final SqlParser.Config PARSER_CONFIG =
+      SqlParser.configBuilder().setLex(PINOT_LEX).setConformance(SqlConformanceEnum.BABEL)
```

Comment:
@npawar I remember you had to relax the SQL parser to accept certain syntax. Was it Babel, or something else?

Reply:
This is the same as the current behavior. Not sure why we need `BABEL` over the default

Reply:
IIRC `BABEL` was able to parse some of the PQL's that the default won't. And the intent was to start off with a mode that makes for a smoother transition.

Reply:
DEFAULT flagged a lot more words as reserved keywords (such as language, module, position etc). BABEL reduced  that set to just the very basic ones (like timestamp, time, date etc).
But like Jackie said, this looks exactly like what it was